testplan/exporters/testing/json/__init__.py

```diff
@@ -68,3 +116,49 @@ def export(self, source):
             self.logger.exporter_info(
                 "Skipping JSON creation for empty report: %s", source.name
             )
+
+    @staticmethod
+    def split_json(data):
+        """Split a single Json into several parts."""
+
+        def split_assertions(entries, assertions):
+            """Remove assertions from report and place them in a dictionary."""
+            for entry in entries:
+                if entry.get("category") == ReportCategories.TESTCASE:
+                    if entry.get("entries"):
+                        # A testcase report should have `parent_uids`
+                        uid = _gen_unique_id(
+                            entry["parent_uids"] + [entry["name"]]
+                        )
+                        assert uid is not None and uid not in assertions
+                        assertions[uid] = entry["entries"]
+                        entry["entries"] = [uid]
+                elif "entries" in entry:
+                    split_assertions(entry["entries"], assertions)
+
+        meta, structure, assertions = data, data.get("entries", []), {}
+        meta["split_report"] = True
+        meta["entries"] = []
+        split_assertions(structure, assertions)
+        return meta, structure, assertions
+
+    @staticmethod
+    def merge_json(meta, structure, assertions):
```

Comment:
Since a `split` method is provided as staticmethod, so a corresponding `merge` method is also provided. A internal script will call both of these methods, help us to deal with JSON report.